%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% baposter Landscape Poster
% LaTeX Template
% Version 1.0 (11/06/13)
%
% baposter Class Created by:
% Brian Amberg (baposter@brian-amberg.de)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
% PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[landscape,a0paper,fontscale=0.285]{baposter} % Adjust font scale/size

\usepackage{graphicx} % Required for including images
\graphicspath{{figs/}} % Directory in which figures are stored

\usepackage{amsmath} % For typesetting math
\usepackage{amssymb} % Adds new symbols to be used in math mode
\usepackage{mathtools}

\usepackage[numbers]{natbib}

\usepackage{booktabs} % Top and bottom rules for tables
\usepackage{enumitem} % Used to reduce itemize/enumerate spacing
\usepackage{palatino} % Use the Palatino font
\usepackage[font=small,labelfont=bf]{caption} % Required for specifying captions to tables and figures

\usepackage{multicol} % Required for multiple columns
\setlength{\columnsep}{1.5em} % Slightly increase the space between columns
\setlength{\columnseprule}{0mm} % No horizontal rule between columns

\newcommand{\compresslist}{ % Define a command to reduce spacing within itemize enumerate environments, this is used right after \begin{itemize} or \begin{enumerate}
\setlength{\itemsep}{1pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
}

\newcommand{\D}{\mathcal{D}}
\newcommand{\E}{\mathbb{E}}
\newcommand{\F}{\mathcal{F}}
\newcommand{\M}{\mathcal{M}}
\newcommand{\R}{\mathbb{R}}
\newcommand{\X}{\mathcal{X}}
\newcommand{\like}{\mathcal{L}}
\newcommand{\prob}{\mathbb{P}}
\newcommand{\1}{\mathbbm{1}}

\definecolor{lightblue}{rgb}{0.145,0.6666,1} % Defines color of content box headers

\begin{document}

\begin{poster}
{
headerborder=closed, % Adds a border around the header of content boxes
colspacing=1em, % Column spacing
bgColorOne=white, % Background color for the gradient on the left side of the poster
bgColorTwo=white, % Background color for the gradient on the right side of the poster
borderColor=lightblue, % Border color
headerColorOne=black, % Background color for the header in the content boxes (left side)
headerColorTwo=lightblue, % Background color for the header in the content boxes (right side)
headerFontColor=white, % Text color for the header text in the content boxes
boxColorOne=white, % Background color of the content boxes
textborder=roundedleft, % Format of the border around content boxes, can be: none, bars, coils, triangles, rectangle, rounded, roundedsmall, roundedright or faded
eyecatcher=true, % Set to false for ignoring the left logo in the title and move the title left
headerheight=0.1\textheight, % Height of the header
headershape=roundedright, % Specify the rounded corner in the content box headers, can be: rectangle, small-rounded, roundedright, roundedleft or rounded
headerfont=\Large\bf\textsc, % Large, bold and sans serif font in the headers of content boxes
%textfont={\setlength{\parindent}{1.5em}}, % Uncomment for paragraph indentation
linewidth=2pt % Width of the border lines around content boxes
}
%-------------------------------------------------------------------------------
% TITLE SECTION
%-------------------------------------------------------------------------------
%
{\includegraphics[height=6.5em]{logo_cal.jpg}} % university/lab logo on left
{\textbf{\LARGE Variance Moderation of Locally Efficient Estimators and
    Supervised Clustering with Applications in High-Dimensional
    Biology\vspace{0.5em}}} % title
{\textsc{Nima S.~Hejazi, Mark J.~van der Laan, \& Alan E.~Hubbard} \hspace{12pt}
  \textit{Graduate Group in Biostatistics}} % Author names and institution
{\includegraphics[height=9em]{logo_sph.jpg}} % university/lab logo on right

%-------------------------------------------------------------------------------
% OVERVIEW
%-------------------------------------------------------------------------------

\headerbox{Overview}{name=overview,column=0,row=0}{

\begin{enumerate}\compresslist
  \item We introduce and implement a general approach for applying variance
    moderation techniques to locally efficient estimators in semiparametric
    statistical models.
  \item The approach allows for such estimators to be utilized for differential
    expression analysis by stabilizing their small-sample properties.
  \item Focusing on targeted maximum likelihood estimation (TMLE), we illustrate
    how the approach generalizes to influence function-based estimators.
  \item We estimate the average treatment effect (ATE) in a study of
    occupational exposure to benzene, identifying \textbf{3280} significant
    genes after controlling the FDR at 5\%.
  \item We further illustrate that our focus on influence function-based
    estimators allows for supervised clustering.
\end{enumerate}

\vspace{0.3em} % When there are two boxes, some whitespace may need to be added
               % if the one on the right has more content
}

%-------------------------------------------------------------------------------
% INTRODUCTION
%-------------------------------------------------------------------------------

\headerbox{Introduction \& Data}
{name=introduction,column=1,row=0,bottomaligned=overview}{

\begin{itemize}\compresslist
  \item With the growing number of methods for measuring biomarkers there arises
    a need for methodologies able to simultaneously analyze multiple kinds of
    exposome data.
  \item Data was generated by the \textbf{Illumina Human Ref-8 BeadChips}
    platform.
  \item There were 125 subjects, for which background characteristics and
    expression measures for $\sim 22,000$ genes were obtained.
  \item Covariates in W were age, sex, and smoking status; all were discretized.
  \item The treatment (A) is degree of Benzene exposure: none, <1ppm, and >5ppm.
  \item The outcome (Y) is a vector of gene expression measures, normalized by
    median.
\end{itemize}

}

%-------------------------------------------------------------------------------
% METHODS 2: Supervised Clustering
%-------------------------------------------------------------------------------
\headerbox{Methodology II: Supervised Distance Matrices}{name=results,column=2,span=2,row=0}{

\begin{multicols}{2}

\begin{center}
\includegraphics[scale=0.3]{topGenesHeatmap}
%\captionof{figure}{raw p-values from applying Limma}
\end{center}

\begin{center}
\includegraphics[scale=0.3]{topGenesHeatmap}
%\captionof{figure}{BH-corrected p-values from applying Limma}
\end{center}

\end{multicols}

%------------------------------------------------

\begin{multicols}{2}

\begin{itemize}\compresslist
\item The raw p-values are bimodally distributed, with a uniform distribution outside 
of the peaks, and clusters near $0$ and $1$. 
\item These raw p-values must be adjusted on account of the $\sim 22,000$ 
simultaneous  tests.
\end{itemize}

\begin{itemize}\compresslist
\item Using the Benjamini-Hochberg procedure to adjust for multiple comparisons 
yields an expected distribution of p-values.
\item \textbf{3280} genes have Benjamini-Hochberg adjusted p-values falling below the 
5\% FDR.
\end{itemize}

\end{multicols}
}

%-------------------------------------------------------------------------------
% REFERENCES
%-------------------------------------------------------------------------------

\headerbox{\small References}{name=references,column=2,above=bottom}{
\renewcommand{\section}[2]{\vskip 0.05em} % remove "References" section title
\tiny{ % Reduce the font size in this block
  \setlength{\bibsep}{0.25pt}
  \bibliographystyle{plainnat}
  \bibliography{2018_ccb}
  \compresslist
  \vspace{-0.7em}
}
}

%-------------------------------------------------------------------------------
% Contact Information
%-------------------------------------------------------------------------------

\headerbox{\small Contact Information}{name=ack,column=3,aligned=references,above=bottom}{
% This block is as tall as the references block
\begin{itemize}
\itemsep0.25pt
  \item \textbf{N.S.~Hejazi}, Ph.D.~student, Group in Biostatistics,
    \textsc{nhejazi@berkeley.edu}
  \item \textbf{M.J.~van der Laan}, Professor of Biostatistics \& Statistics,
    \textsc{laan@berkeley.edu}
  \item \textbf{A.E.~Hubbard}: Professor of Biostatistics,
    \textsc{hubbard@berkeley.edu}
\end{itemize}
}

%-------------------------------------------------------------------------------
% CONCLUSION
%-------------------------------------------------------------------------------

\headerbox{Results \& Discussion}
{name=conclusion,column=2,span=2,row=0,below=results,above=references}{

\vspace{0.25em}

\begin{multicols}{2}

\begin{center}
\vspace*{-0.5cm}
\includegraphics[scale=0.37]{topGenesHeatmap}
%\captionof{figure}{This figure demonstrates...}
\end{center}

\begin{center}
%\vspace*{-0.49cm}  % CHANGE THIS TO ALIGN IMAGES
\includegraphics[scale=0.39]{topGenesHeatmap}
%\captionof{figure}{This figure demonstrates...}
\end{center}

\vspace{-15pt}

\begin{itemize}
  \itemsep0pt
  \item The heatmap visualizes the ATE difference induced by benzene exposure.
  \item The x-axis shows the 125 subjects, while the y-axis shows the top 25
    genes showing highest differential ATE (based on BH adjusted p-values).
  \item Blue indicates a depression in the ATE, while red indicates an increase
    in the ATE, based on exposure to the maximal level of benzene as opposed to
    not.\\
\end{itemize}

\end{multicols}
}


%-------------------------------------------------------------------------------
% METHODS
%-------------------------------------------------------------------------------

\headerbox{Methodology I: Variance Moderation \& Asymptotic Linearity}{name=method,column=0,span=2,below=overview,bottomaligned=references}{
% This block's bottom aligns with the bottom of the conclusion block

\begin{itemize}
  \itemsep0.50pt
  \item Let observed data $O = (W, A, Y) \sim P_0 \in \M$, where $W$ represents
      potential baseline confounders, $A$ the exposure of interest, and
      $Y = ({Y_b}, b = 1, \dots, B)$ a vector of potential biomarkers.
  \item We consider, as an example, the \textit{average treatment effect} (ATE),
      as the causal parameter of interest, which is identified by the observed
      data parameter:
      \begin{equation}\label{ate}
        \Psi_b(P_0) = \E_W[ Q_0^b(A = 1, W) - Q_0^b(A = 0, W)],
      \end{equation}
        where $Q_0^b(A, W) \equiv \E_{P_0}(Y_b \mid A, W)$ and may be estimated
        via \textit{ensemble machine learning}
        \cite{vdl2007super,breiman1996stacked,wolpert1992stacked}.
  %\item To estimate $\Psi$ using the construction from Eqn.~\ref{ate}, a plug-in
    %estimator may be constructed
    %\begin{equation}
      %\Psi_b(P_n) = \frac{1}{n} \sum_{i = 1}^n Q_n^b(A_i = 1, W_i) -
      %Q_n^b(A_i = 0, W_i),
    %\end{equation}
      %where ${Q_n}^b$ is an initial estimate of ${Q_0}^b$ constructed via
      %\textit{ensemble machine learning} \cite{vdl2007super,breiman1996stacked,
      %wolpert1992stacked}.
  \item Similarly to the estimator $\hat{\beta}$ in a linear model, the
      $\Psi_b(P_n)$ is \textbf{asymptotically linear} (for $\Psi_b$)
      \cite{tsiatis2007semiparametric,vdl2011targeted}:
      \begin{equation}\label{asymp_lin}
        \sqrt{n} (\Psi_b(P_n) - \Psi_b(P_0)) = \frac{1}{\sqrt{n}} \sum_{i=1}^n
        D_b(O_i) + o_p(1).
      \end{equation}
  \item $\Psi_b$ has (efficient) influence function, relative to the
      nonparametric model $\M$:
      \begin{equation}\label{eif_ate}
        D_b(P_0)(o) = \left(\frac{I(a = 1)}{g(1 \mid w)} -
        \frac{I(a = 0)}{g(0 \mid W)}\right) \cdot [y_b - Q_0^b(a, w)] +
        Q_0^b(1, w) - Q_0^b(0, w) - \Psi_b(P_0)(o).
      \end{equation}
  \item The moderated t-statistic \cite{smyth2004linear,smyth2005limma} may be
    applied readily to asymptotically linear estimators:
    $\tilde{t}_b = \frac{\sqrt{n}(\Psi_b(P_n) - \psi_0)}{S_b(D_{b,n})}$, where
    $\tilde{S}_{b,n}^2 = \frac{d_0S_0^2 + d_bS_b^2(D_{b,n})}{d_0 + d_b}$ where
    $d_b$ is the degrees of freedom for the $b^{th}$ biomarker, $d_0$ is the
    degrees of freedom for the remaining $(B - 1)$ biomarkers, $S_b$ is the
    standard deviation for the $b^{th}$ biomarker and $S_0$ is the common
    standard deviation across all biomarkers.
\end{itemize}

}

%-------------------------------------------------------------------------------

\end{poster}

\end{document}
